src/tests/ftest/daos_test/suite.py

```diff
@@ -408,3 +408,17 @@ def test_daos_dedup(self):
         :avocado: tags=daos_test,daos_core_test,test_daos_dedup
         """
         self.run_subtest()
+    def test_daos_upgrade(self):
+        """Jira ID: DAOS-1568
+
+        Test Description:
+            Run daos_test -P
+
+        Use cases:
+            Core tests for daos_test
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,ib2,medium
```

Comment:
ib2 is no longer used
```suggestion
        :avocado: tags=hw,medium
```

Reply:
sure, thanks